packages/protocol/scripts/truffle/deploy_release_contracts.ts

```diff
@@ -167,21 +162,11 @@ async function handleGrant(config: ReleaseGoldConfig, currGrant: number) {
       from: fromAddress,
     },
   ])
-  const initializerAbiRG = checkAndReturnInitializationAbi(ReleaseGold, 'ReleaseGold')
-  const transferImplOwnershipAbiRG = checkAndReturnTransferOwnershipAbi(ReleaseGold, 'ReleaseGold')
   console.info('  Deploying ReleaseGoldProxy...')
   const releaseGoldProxy = await retryTx(ReleaseGoldProxy.new, [{ from: fromAddress }])
   console.info('  Deploying ReleaseGold...')
   const releaseGoldInstance = await retryTx(ReleaseGold.new, [{ from: fromAddress }])
 
-  console.info('Initializing ReleaseGold...')
-  await initializeRGImplementation(
```

Comment:
We can remove all this now because this general solution with the Initializer constructor makes it irrelevant.